codingTest_kjy/flag에따라다른값반환하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(a, b, flag) { 
+    var answer = 0;              
+    if(flag){
+    return answer = a + b
+    }
+    return answer = a - b
```

Comment:
```suggestion
    return a + b
    }
    return a - b
```
answer에 담지 않고 이렇게 바로 해도 될 것 같아요.